js/storage.js

```diff
@@ -0,0 +1,22 @@
+const form1 = document.querySelector('#contact');
+form1.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const obj = {
+    fullName: document.querySelector('#name').value,
+    emailAdress: document.querySelector('#email').value,
+    message: document.querySelector('#message').value,
```

Comment:
- [ ] Please make sure that you have input fields in the HTML with these IDs. Since you are selecting HTML elements by their IDs you should have an HTML element with the same ID you are using here. This is your form at the moment. ✍️

![image](https://user-images.githubusercontent.com/17243325/210787329-fe483a88-0533-45e1-8406-8712191086aa.png)

Reply:
Sorry I forgot to push (upload) my index.html for this project